test/ibmq/test_ibmq_job_attributes.py

```diff
@@ -62,10 +62,9 @@ def test_running_job_properties(self, provider, backend):  # pylint: disable=unu
         job = backend.run(qobj)
         _ = job.properties()
 
-    @requires_provider
-    def test_job_name_backend(self, provider):
+    @requires_device
+    def test_job_name_device(self, backend):
```

Comment:
There isn't really a need to run the job name test on both real device and simulator, as job names are tracked by the api, not the backend.